defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+gocd_common_user: go
+gocd_common_userid: 5000
+gocd_common_group: go
+gocd_common_groupid: 5000
+gocd_common_debian_repository: http://dl.bintray.com/gocd/gocd-deb/
+gocd_common_configure_ssh: false
+gocd_common_ssh_known_domain: github.com
+gocd_common_java_versions:
+  - oracle-java8-installer
+  - oracle-java8-set-default
```

Comment:
Any package (preferably along with its dependencies) need to have specific version pinned.